### ISSUE TITLE
Windows Security Trojen alert

### ISSUE THREAD

**Issue by kerimugurdonmez** [PS3Xploit/PS3xploit-resigner#42]
Trojan:Win32/Occamy.B
Is this normal? Why is Windows Defender alarming?
![Ekran Alıntısı](https://user-images.githubusercontent.com/28684471/76503471-08281b80-6457-11ea-8896-ac0d88fcee67.PNG)

**Comment by silver178** [comment]
Yes it's normal and I shall tell you why. Assuming you're on windows 11 Microsoft updated windows antivirus so now it detects a lot more stuff as kind of false positives. I definitely can and will be wrong but we'll see on that but I've seen it happen quite a bit. Hope this helps and clears things up.
